Sources/Starknet/Providers/StarknetProvider/JsonRpcParams.swift

```diff
@@ -162,3 +162,61 @@ struct SimulateTransactionsParams: Encodable {
         case simulationFlags = "simulation_flags"
     }
 }
+
+enum EncodableParams {
+    case getNonceParams(GetNonceParams)
+    case addInvokeTransactionParams(AddInvokeTransactionParams)
+    case wrappedExecutableParams(WrappedExecutableTransaction)
+    case emptySequenceParams(EmptySequence)
+    case emptyParams(EmptyParams)
+    case callParams(CallParams)
+    case estimateFeeParams(EstimateFeeParams)
+    case estimateMessageFeeParams(EstimateMessageFeeParams)
+    case addDeployAccountTransactionParams(AddDeployAccountTransactionParams)
+    case getClassHashAtParams(GetClassHashAtParams)
+    case getEventsPayload(GetEventsPayload)
+    case getTransactionByHashParams(GetTransactionByHashParams)
+    case getTransactionByBlockIdAndIndex(GetTransactionByBlockIdAndIndex)
+    case getTransactionReceiptPayload(GetTransactionReceiptPayload)
+    case getTransactionStatusPayload(GetTransactionStatusPayload)
+    case simulateTransactionsParams(SimulateTransactionsParams)
```

Comment:
Seems to me the naming of cases is a bit inconsistent. 

Some have `Params` postfix, some `Payload`, some none. We normally don't need to include postfix that is already present in enum name, so let's drop all the postfixes:
```suggestion
    case getNonce(GetNonceParams)
    case addInvokeTransaction(AddInvokeTransactionParams)
    case wrappedExecutableTransaction(WrappedExecutableTransaction)
    case emptySequence(EmptySequence)
    case empty(EmptyParams)
    case call(CallParams)
    case estimateFee(EstimateFeeParams)
    case estimateMessageFee(EstimateMessageFeeParams)
    case addDeployAccountTransaction(AddDeployAccountTransactionParams)
    case getClassHashAt(GetClassHashAtParams)
    case getEvents(GetEventsPayload)
    case getTransactionByHash(GetTransactionByHashParams)
    case getTransactionByBlockIdAndIndex(GetTransactionByBlockIdAndIndex)
    case getTransactionReceipt(GetTransactionReceiptPayload)
    case getTransactionStatus(GetTransactionStatusPayload)
    case simulateTransactions(SimulateTransactionsParams)
```